text/0000-kpack-donation-to-cnb.md

```diff
@@ -0,0 +1,355 @@
+
+How do migrate roles and responsibilities into the CNB governance process?
+
+Currently the [CNB Platform Team](https://github.com/buildpacks/community/blob/main/TEAMS.md#Platform-Team) already has a **team lead** assigned and, by definition, each team can have only one **team lead**. In order to provide the current [kpack](https://github.com/pivotal/kpack/) team with the same accountability for the migrated repositories the proposal is to follow the guidelines describe on the [Component Maintainer Role RFC](https://github.com/buildpacks/rfcs/pull/234)
+
+#### RFC process
+
+Once the migration is completed, [kpack](https://github.com/pivotal/kpack/) will follow the [RFC process](https://github.com/buildpacks/rfcs) and [RFC template](https://github.com/buildpacks/rfcs/blob/main/0000-template.md) stablished in CNB project for any new RFC created in the project.
+
+##### Existing RFC
+
+- **Open**: Currently there are less that 10 [open RFCs](https://github.com/pivotal/kpack/pulls?q=is%3Apr+label%3ARFC+is%3Aopen) (some of them opened 2 years ago) in [kpack](https://github.com/pivotal/kpack/) repository.
+  - The proposal is to suggest the [kpack](https://github.com/pivotal/kpack/) maintainers to:
+    - Triage those RFCs an update their status before the donation.
+    - Co-ordinate the announcement of the donation to the RFCs authors and explain them the strategy after the migration (next section)
+    - After the donation, any open RFCs in [kpack](https://github.com/pivotal/kpack/) repository should be closed
+    - The RFC author should create a new RFC in the CNB RFC [repository](https://github.com/buildpacks/rfcs) and follow the CNB [RFC process](https://github.com/buildpacks/rfcs)
+
+- **Closed**: For historical purpose, we will keep those RFC in the repository.
+
+#### Slack channel
+
+The proposals are:
+ - `keep` the [kpack](https://github.com/pivotal/kpack/) slack instance from the [Kubernetes slack instance](https://kubernetes.slack.com/channels/kpack), as [kpack](https://github.com/pivotal/kpack/) is a Kubernetes native application most of their users already use [Kubernetes slack instance](https://kubernetes.slack.com/channels/kpack) for communication.
+ - `create` a new channel in the [CNCF slack instance](https://slack.cncf.io/), this will bring the two communities (kpack and CNB) together
+
+[kpack](https://github.com/pivotal/kpack/) maintainers should include the notification of the new channel in the announcement of the donation.
+
+[Platform maintainers](https://github.com/buildpacks/community/blob/main/TEAMS.md#maintainers-1) will have to request or create the new slack channel with the following name: **buildpacks-kpack** (which will be defined as the preferred channel to be used).
+
+# Risks
```

Comment:
Is VMware the only core contributor to kpack?

If so, then a reduction in investment from VMware would be a huge risk. The CNB proejct would have to either sunset kpack, or find investment from the community.

Reply:
I added the risk